### PR TITLE
[COD-39] Connect backend and frontend for create cource

### PR DESCRIPTION
**Pull request by SilaEwGenii**
[JIRA ticket](https://legolas.atlassian.net/browse/COD-39)
Frontend
When creating a course, a message is entered about the creation of the course, which can be closed by clicking on the cross in the right corner of the message window.
Снимок экрана 2023-03-11 в 13 54 03

If you enter an incorrect value in the name field, the form is highlighted in red and displays an error pop-up message.
Снимок экрана 2023-03-11 в 14 13 00

Backend
1. Changed error output from error.full_messages  to error.messages in controller courses and lessons.
2. Сreated a connection between frontend and backend.

### CODE REVIEW

**Review by Set27 — CHANGES_REQUESTED** [review]
Add i18n
"COD 39" to "COD-39"

**Comment by SilaEwGenii** [comment]
> Добавить i18n "COD 39" в "COD-39"

sorry, thanks for the comment

**Review by npupko — CHANGES_REQUESTED** [review]
Let's add errors handler just in case

**Comment by SilaEwGenii** [comment]
@Set27 , @npupko 
Good afternoon, the comments have been eliminated, I ask you to re-PR


1. [Added localization for the course creation page]

Снимок экрана 2023-03-15 в 00 42 44


2. [Display a course creation message]

Снимок экрана 2023-03-15 в 00 43 14


3. [Error message display]

Снимок экрана 2023-03-15 в 00 43 39

**Review by Set27 — APPROVED** [review]
lgtm

File: frontend/src/lib/i18n/locales/en.json [hunk 1]
```diff
@@ -26,6 +26,16 @@
...
       "forgot": "Forgot your password?",
       "sign_in_button": "Sign in"
+    },
+    "create_course": {
+      "errors": {
+        "name": "Course name can't be blank"
+      },
+      "title": "Create Course",
+      "input_label": "Enter course name",
+      "create_button": "Create Course",
```
Comment (npupko) [inline frontend/src/lib/i18n/locales/en.json:36]: Let's be consistent and use same register for course everywhere.
I.e. if it's (Enter **course** name) than it should also be "Create **course**" (not Create **Course**), etc.
Downcase for course is ok there

**Review by npupko — APPROVED** [review]
LGTM with small fixes

**Pull request merged**